backend/tracim_backend/exceptions.py

```diff
@@ -283,3 +283,7 @@ class UnavailablePreview(TracimException):
 
 class EmptyNotificationError(TracimException):
     pass
+
+
+class ContentClosed(TracimException):
```

Comment:
The concept is not "content is closed so it is not editable" but "the content is in a read-only status". For example, an archived content or a deleted content is also not editable. So the `ContentClosed` exception should be renamed to something like `ContentInReadOnlyState`.